tekton/resources/nightly-tests/triggers-deploy-test-s390x-template.yaml

```diff
@@ -0,0 +1,201 @@
+          runAfter: [create-k8s-cluster]
+          taskRef:
+            name: deploy-tekton-project-nightly
+          workspaces:
+          - name: k8s-shared
+            workspace: k8s-shared
+          params:
+          - name: package
+            value: github.com/tektoncd/pipeline
+          - name: container-registry
+            value: $(params.container-registry)
+          - name: target-arch
+            value: $(params.target-arch)
+          resources:
+            inputs:
+            - name: tekton-project-source
+              resource: tekton-pipeline-source
+        - name: deploy-triggers
+          runAfter: [deploy-pipeline]
+          taskRef:
+            name: deploy-tekton-project-nightly
+          workspaces:
+          - name: k8s-shared
+            workspace: k8s-shared
+          params:
+          - name: package
+            value: $(params.package)
+          - name: container-registry
+            value: $(params.container-registry)
+          - name: target-arch
+            value: $(params.target-arch)
+          resources:
+            inputs:
+            - name: tekton-project-source
+              resource: tekton-triggers-source
+        - name: e2e-test-triggers
+          runAfter: [deploy-triggers]
+          taskRef:
+            name: test-e2e-tekton-component
+          workspaces:
+          - name: k8s-shared
+            workspace: k8s-shared
+          retries: 2
+          params:
+          - name: package
+            value: $(params.package)
+          - name: container-registry
+            value: $(params.container-registry)
+          - name: target-arch
+            value: $(params.target-arch)
+          resources:
+            inputs:
+            - name: plumbing-source
+              resource: plumbing-source
+            - name: tekton-project-source
+              resource: tekton-triggers-source
+        finally:
+        - name: delete-k8s-cluster
+          taskRef:
+            name: create-delete-k8s-cluster-$(tt.params.targetArch)
+          workspaces:
+          - name: k8s-shared
+            workspace: k8s-shared
+          - name: ssh-secret
+            workspace: ssh-secret
+          params:
+          - name: remote-host
+            value: $(params.remote-host)
+          - name: remote-port
+            value: $(params.remote-port)
+          - name: remote-user
+            value: $(params.remote-user)
+          - name: action
+            value: delete
+      params:
+      - name: package
+        value: github.com/tektoncd/triggers
+      - name: container-registry
+        value: $(tt.params.containerRegistry)
+      - name: target-arch
+        value: $(tt.params.targetArch)
+      - name: remote-host
+        value: $(tt.params.remoteHost)
+      - name: remote-port
+        value: $(tt.params.remotePort)
+      - name: remote-user
+        value: $(tt.params.remoteUser)
+      resources:
+      - name: tekton-pipeline-source
```

Comment:
I'd rather not continue adding new pipeline resources, but as mentioned earlier I'm ok with it for now.
I'd love to see at some point a follow up that removes them 🙏

Reply:
yeah, I definitely remember about moving to git-clone usage. Could you please install this task to bastion-z namespace? and with the next PR I will use it for all s390x pipelines.

Reply:
+1 on @afrittoli comment